modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/structure.rb

```diff
@@ -82,22 +80,37 @@ def form_fill
             # NOT YET SUPPORTED
           }
 
-          fill_contestable_issues!(options)
+          fill_contestable_issues_dates!(options)
         end
         # rubocop:enable Metrics/MethodLength
 
         def insert_overlaid_pages(form_fill_path)
-          form_fill_path
+          pdftk = PdfForms.new(Settings.binaries.pdftk)
+          temp_path = "/#{::Common::FileHelpers.random_file_path}.pdf"
+          output_path = "/tmp/HLRv2-#{higher_level_review.id}-overlaid-form-fill-tmp.pdf"
+
+          Prawn::Document.generate(temp_path) do |pdf|
```

Comment:
can this be a private method? `whiteout_boxes` or something, that returns `temp_path`?